config/crd/bases/kafka.nais.io_aivenapplications.yaml

```diff
@@ -0,0 +1,107 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.5
+  creationTimestamp: null
+  name: aivenapplications.kafka.nais.io
```

Comment:
The name and namespace of the AivenApplication will be the same name and namespace of the nais app that "owns" it.

Which topic should we put in the label? The first one that saw this application? The last one? The last one that was edited?

Early on in the ADR process we had the idea of maintaining a list of topics for each AivenApplication, but realized that we had no way of maintaining that list when an application loses access to a topic. Since this was a nice-to-have feature, we agreed that it was simpler to drop the feature, since it would be difficult to make a list that could be properly maintained.

Having a label that points to a "randomly" selected topic that may or may not be related to this AivenApplications doesn't seem that useful to me.